Tmain/readtags-qualifier-begin.d/run.sh

```diff
@@ -20,8 +20,8 @@ fi
 
 run_readtags()
 {
-	local fexpr=$1
-	local action=$2
+	local fexpr="$1"
+	local action="$2"
```

Comment:
Is this for fixing a bug or just for prevention?

```
[jet@living]/tmp/ctags/universal-ctags-5.9.0/_build/sub/Tmain/lang-sequel.d% dash
$ x='a b'
$ y=$x         
$ echo $y
a b
```

I think this change is for prevention.

Reply:
It was for a bug fix.  I saw errors on this test (and epoch-field.d, too). But I cannot reproduce them.
It seems that something was wrong with me.

I misunderstood that shell expansions rule.  Shell expansions are done after variable assignments.

I've revert the change.